packages/blue-sdk/src/preLiquidation.ts

```diff
@@ -6,71 +6,81 @@ export const defaultPreLiquidationParamsRegistry = new Map([
   [
     parseEther("0.385"),
     {
-      preLltv: 30_1514568055515563n,
+      preLltv: 301514568055515563n,
       preLCF1: 22637943984157107n,
-      preLCF2: 34_9673199983645648n,
+      preLCF2: 349673199983645648n,
       preLIF1: parseEther("1.15"),
       preLIF2: parseEther("1.15"),
     },
   ],
   [
     parseEther("0.625"),
     {
-      preLltv: 56_2591950487445723n,
+      preLltv: 562591950487445723n,
       preLCF1: 7543182567291709n,
-      preLCF2: 27_9542312587328718n,
-      preLIF1: 1_126760563380281690n,
-      preLIF2: 1_126760563380281690n,
+      preLCF2: 279542312587328718n,
+      preLIF1: 1126760563380281690n,
+      preLIF2: 1126760563380281690n,
     },
   ],
   [
     parseEther("0.77"),
     {
-      preLltv: 72_7366070175296029n,
+      preLltv: 727366070175296029n,
       preLCF1: 3706417131700377n,
-      preLCF2: 25_6643181309902852n,
-      preLIF1: 1_074113856068743286n,
-      preLIF2: 1_074113856068743286n,
+      preLCF2: 256643181309902852n,
+      preLIF1: 1074113856068743286n,
+      preLIF2: 1074113856068743286n,
     },
   ],
   [
     parseEther("0.86"),
     {
-      preLltv: 83_2603694978499652n,
+      preLltv: 832603694978499652n,
       preLCF1: 2001493508968667n,
-      preLCF2: 24_5311807032632372n,
-      preLIF1: 1_043841336116910229n,
-      preLIF2: 1_043841336116910229n,
+      preLCF2: 245311807032632372n,
+      preLIF1: 1043841336116910229n,
+      preLIF2: 1043841336116910229n,
     },
   ],
   [
     parseEther("0.915"),
     {
-      preLltv: 89_7868776651447149n,
+      preLltv: 897868776651447149n,
       preLCF1: 1135586186384195n,
-      preLCF2: 23_9205538157954887n,
-      preLIF1: 1_026167265264238070n,
-      preLIF2: 1_026167265264238070n,
+      preLCF2: 239205538157954887n,
+      preLIF1: 1026167265264238070n,
+      preLIF2: 1026167265264238070n,
+    },
+  ],
+  [
+    parseEther("0.945"),
+    {
+      preLltv: 933746617913300027n,
+      preLCF1: 709220796660800n,
+      preLCF2: 236098907251355946n,
+      preLIF1: 1016776817488561260n,
+      preLIF2: 1016776817488561260n,
```

Comment:
```suggestion
    parseEther("0.945"),
    {
      preLltv: 93_3746617913300027n,
      preLCF1: 709220796660800n,
      preLCF2: 23_6098907251355946n,
      preLIF1: 1_016776817488561260n,
      preLIF2: 1_016776817488561260n,
```

Oups, this change is also the reason for this PR, just curious why you didn't like the syntax?

Reply:
It didn't seem useful to me but now I understand you use it to distinguish unit and decimals, I'll change it